doc-content/enterprise-only/openshift/readwritemany-proc.adoc

```diff
@@ -0,0 +1,69 @@
+[id='readwritemany-proc']
+= Provisioning persistent volumes with ReadWriteMany access mode using NFS
+
+:rwm-authoring!:
+:rwm-bcmonitoring!:
+
+:rwm-gluster!:
+ifeval::["{context}"=="openshift-operator"]
+:rwm-authoring:
+:rwm-bcmonitoring:
+endif::[]
+ifeval::["{context}"=="openshift-authoring"]
+:rwm-authoring:
+:rwm-gluster:
+endif::[]
+ifeval::["{context}"=="openshift-authoring-managed"]
+:rwm-authoring:
+:rwm-gluster:
+endif::[]
+
+
+ifdef::rwm-authoring[]
+ifdef::PAM[]
+If you want to deploy 
+ifdef::rwm-bcmonitoring[]
+{CENTRAL} Monitoring, high-availability {CENTRAL},
+endif::rwm-bcmonitoring[]
+ifndef::rwm-bcmonitoring[]
+high-availability {CENTRAL}
+endif::rwm-bcmonitoring[]
+or any {KIE_SERVERS} that use the H2 database, which is the default setting for a non-high-availability authoring environment, your environment must provision persistent volumes with ReadWriteMany access mode.
+endif::PAM[]
+ifdef::DM[]
+If you want to deploy high-availability {CENTRAL}, your environment must provision persistent volumes with ReadWriteMany access mode.
+endif::DM[]
+
+ifdef::rwm-gluster[]
+[NOTE]
+====
+If you want to deploy a high-availability authoring environment, for optimal performance and reliability provision persistent volumes using GlusterFS. Configure the GlusterFS storage class as described in <<glusterfs-reconfig-proc>>.
```

Comment:
Trivial, but suggest "... and reliability, provision ..." I know it chops it up more, but helps parse better for easier reading processing.